RELEASE.md

```diff
@@ -0,0 +1,16 @@
+Release type: minor
+
+# New Features
+Added support for sanic webserver.
+
+# Changelog
+Deprecated the following type annotation,
+```python
+from strawberry.schema.base import ExecutionResult # Deprecated
+```
```

Comment:
```suggestion
`ExecutionResult` was erroneously defined twice in the repository. The entry in `strawberry.schema.base` has been removed. If you were using it, switch to using
`strawberry.types.execution.ExecutionResult` instead.
```

Reply:
It can't be deprecated if it's removed 🙂

Reply:
Haha, fair